gitbook/api-documentation/readme.md

```diff
@@ -4,6 +4,10 @@ As explained on the [Affordances][a] page, each and every resource object within
 with an `apiDoumentation` getter. This API Documentation is fetched from the web by following the HTTP `Link`
 header as specified by [Hydra][api-doc-spec].
 
+{% hint style="tip" %}
+ In order for Alcaeus to respect the `Link` header the `Access-Control-Expose-Headers` header has to be set   as well.
```

Comment:
This is only true when CORS is in effect. Could you elaborate more, that it only applies to browser environment when requesting from a different domain?

Reply:
It now mentions CORS, the heads up should be enough as a pointer for further research.